utils/arweave.js

```diff
@@ -79,7 +88,12 @@ async function submitToArweave(file, ipfsHash = null) {
     tx.addTag(IPFS_CONSTRAINT_KEY, IPFS_CONSTRAINT);
   }
   await arweave.transactions.sign(tx, jwk);
-  await arweave.transactions.post(tx);
+  const uploader = await arweave.transactions.getUploader(tx, buffer);
+  while (!uploader.isComplete) {
+    // eslint-disable-next-line no-await-in-loop
+    await uploader.uploadChunk();
+    console.log(`${uploader.pctComplete}% complete, ${uploader.uploadedChunks}/${uploader.totalChunks}`);
```

Comment:
Can we show file name/hash/id so that we know which file progress we are looking at?